### PR TITLE
added invisibility potion

### PR DESCRIPTION
**Pull request by ostapyrih**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,6 @@ The following potions are present in this module:
 * Resist Poison - Reduces the effects of poison statuses by 2 when consumed for 15 seconds.
 * Swim Speed - Doubles the user's swim speed for 10 seconds.
 * Walk Speed - Doubles the user's walk speed for 10 seconds.
-* Explosive - When the player uses this on an object, it triggers an explosion.
```
Comment (rzats) [inline README.md:24]: This shouldn't be removed.

File: module.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "id" : "Potions",
     "version" : "0.2.1-SNAPSHOT",
-    "author" : "bi0hax, XTariq, shardulc, S-ulphuric, kallentu",
+    "author" : "bi0hax, XTariq, shardulc, apaukraine",
```
Comment (rzats) [inline module.txt:4]: These shouldn't be removed.

**Review by rzats — CHANGES_REQUESTED** [review]
The potion has to actually make the player invisible :) To do that, you'll need to declare an effect - INVIS - in ([PotionCommonEffects](https://github.com/Terasology/Potions/blob/master/src/main/java/org/terasology/potions/PotionCommonEffects.java)) and code its' functionality (in [DrinkPotionAuthoritySystem](https://github.com/Terasology/Potions/blob/master/src/main/java/org/terasology/potions/system/DrinkPotionAuthoritySystem.java)).

Also, the art is a little odd - that's an invisibility _cloak_, not a potion! I'd change it to a spectral-looking bottle.

**Comment by smsunarto** [comment]
Closed for inactivity - @apaukraine Please update your fork and re-open when you are willing to continue this PR.

**Pull request closed without merging**